packages/terra-date-picker/src/terra-dev-site/test/date-picker/calendar-filter/CalendarFilterFilterDates.test.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import moment from 'moment';
+import CalendarFilter from '../../../../CalendarFilter';
+
+const isWeekday = (date) => {
```

Comment:
A recent change was made to display the Today button if today is considered a disabled day. You would want to add this (https://github.com/cerner/terra-framework/blob/master/packages/terra-date-picker/src/terra-dev-site/test/date-picker/DatePickerFilterDates.test.jsx#L12-L15) to make sure that today's date does not fall on a weekend. Otherwise, the Today button on the screenshot would mismatch.

Reply:
added it up here